### ISSUE TITLE
Highlight active window for quicker interactions

### ISSUE THREAD

**Issue by benz145** [mzomparelli/zVirtualDesktop#201]
In the default (Windows) version of desktop view, all of the windows on the current monitor animate into shrunken versions of themselves, which makes it easy to follow the currently active window (which, more often than not, is the one you want to act upon immediately after triggering the desktop view). 

In the ZVD version of desktop view, the view appears instantly with no direct connection between the actual windows and their thumbnails. This makes it easy to lose track of the active window, which makes acting on it slower. It seems the default behavior is to put the active window at the top of the stack (?), but a clearer visual hierarchy would make this more clear.

In lieu of an animation like Windows does it (which I think would be the best approach, but probably beyond the scope of this request currently), it would be good to more clearly highlight the active window to make it quicker to act upon. It would also be nice if the window thumbnails could be dragged directly, instead of just instantly popping between desktop columns as you drag them.

More clearly highlighting the active desktop would also be beneficial for the same reasons (faster visual targeting) -- I see you can customize the header colors for the active desktop, but a more clear indicator would be ideal, perhaps a highlight around the entire border would be a good approach, or coloring the entire background. A change to the default active desktop header colors (to make it clearly standout as the active desktop) would probably be a quick and easy starting point.

**Comment by mzomparelli** [comment]
This is good feedback. I'll see what I can do.

In case you aren't aware, if you enable the window timers then it should highlight the timer for the active running window timer. This isn't exactly what you are looking for, however it could help you locate the active window while I try to implement your idea.

**Comment by benz145** [comment]
Thanks, I did notice the active timer gets colored, but it’s too subtle to
create a quickly perceived visual hierarchy.

I appreciate you looking into these requests!

**Comment by mzomparelli** [comment]
I've handled the popping from one column to another, however currently I'm unable to get a thumbnail preview when dragging. It seems ok for now and I'll continue figuring out the other stuff.

**Comment by benz145** [comment]
I noticed the other day when I was playing a fullscreen game that the thumbnail in Desktop View was not only an active view of the game (I could see the game animating in the thumbnail), but when I dragged the thumbnail it didn't go blank like the others -- I continued to see the game in the thumbnail. Maybe this offers some sort of hint for how thumbnail dragging might be achieved without having them go blank?

Edit: Just noticed thumbnail dragging also works correctly with Photoshop and Steam. Maybe GPU-accelerated applications work correctly by default?

**Comment by mzomparelli** [comment]
I'm aware of some windows showing black instead of a thumbnail. I'm looking for a solution to this. Personally, it's not a big deal to me,  however it could be seen as unprofessional.

**Comment by benz145** [comment]
Yeah it isn't a program-breaking feature or anything, just tossing out feedback as I see it : )
